### PR TITLE
Small changes and correction to the introduction

### PR DESCRIPTION
**Pull request by samwalle**

### CODE REVIEW

**Review by OskarSundberg — APPROVED** [review]
LGTM

**Review by JoelScarinius — CHANGES_REQUESTED** [review]
I like the changes to the text.

There are some spelling errors that would be awesome if you fix and then push again. 

If you don't use a spellchecker I can show some nice tools to help you out.

File: report/Introduction.tex [hunk 1]
```diff
@@ -5,13 +5,13 @@
 \textcolor{red}{The introduction should briefly introduce the assignment and its purpose.}
 
 This report addresses an investigation for optimizing an elevators route to improve the efficiency of picking up and
-delivering passengers to their desired floors in the leats amount avrage waiting time.
-In large buildings with a lot of floors, efficient elevators are crucial for managing trafick and must serve every one in a reasenable time.
+delivering passengers to their desired floors in the least amount average waiting time.
+In large buildings with a lot of floors, efficient elevators are crucial for managing traffick and must serve every one in a reasonable time.
```
Comment (Linkan42) [inline report/Introduction.tex:9]: actually, it is spelled "traffic"

**Review by Linkan42 — CHANGES_REQUESTED** [review]

**Comment by OskarSundberg** [comment]
Closing this PR because I'm going to fix more stuff and to avoid more trouble.

**Pull request closed without merging**